### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by developertariq**
- ### Form created with HTML5 validations
- ### Followed the best practices for HTML & CSS
- ### Used Flexbox to lay out the elements on the page
- ### Following thee template 2 of Figma 

contactme

- ### Ensured input validation
- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.

- ### Collected data by using the Formspree service
mail-me

formspree-data

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -129,5 +129,26 @@ <h2>About me</h2>
...
+            <h2 id="contact-para">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/meqdpjbw" method="post">
+                <fieldset>
+                    <input type="text" id="name" name="fullname" maxlength="30" placeholder="Full name" required />
+                    <input type="email" id="email" name="email" placeholder="Email address" required />
+                    <textarea id="msg" class="form-msg" name="msg" maxlength="500" required>Enter text here
+                    </textarea>
+                    <input type="submit" id="submit" value="Get in touch"> 
+                </fieldset>            
+            </form>
```
Comment (Nemwel-Boniface) [inline index.html:143]: - [ ] Good job styling your contact section. However, it seems like the contact form and the preceding text are overlapping each other and the design looks a bit flawed. Kindly look into why this is happening and solve it.

**This is how I am seeing the section:**

image

**This is the desired design:**

image

File: index.html [hunk 2]
```diff
@@ -129,5 +129,26 @@ <h2>About me</h2>
...
+            <ul id="contact-social-media">
+                <li><a href="https://github.com/"><img src="./icons/github.svg"></a></li>
+                <li><a href="https://www.linkedin.com/"><img src="./icons/Linkedin.svg"></a></li>
+                <li><a href="https://angel.co/"><img src="./icons/Angelist.svg"></a></li>
+                <li><a href="https://twitter.com/"><img src="./icons/twitter.svg"></a></li>
+                <li><a href="https://medium.com/"><img src="./icons/medium.svg"></a></li>
+            </ul>
```
Comment (Nemwel-Boniface) [inline index.html:150]: - [ ] Your social media icons look very faint and are almost not visible at all.Kindly consider changing the current shade and use the one provided in Figma.

**This is how I am seeing your design:**

image

**This is the desired design:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @developertariq  and @Anand8317   :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on the portfolio so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @developertariq ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**